src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The href attribute in the link tag should point to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be used in HTML.